src/main/java/ro/hiringsystem/repository/CandidateUserRepository.java

```diff
@@ -13,7 +13,8 @@ public interface CandidateUserRepository extends JpaRepository<CandidateUser, UU
     @Query("SELECT u FROM CandidateUser u WHERE u.id = :id")
     Optional<CandidateUser> findById(UUID id);
 
-    @Query("SELECT u FROM CandidateUser u WHERE :mail MEMBER OF u.mailList")
+    //modified for testing the authentication system
+    @Query("SELECT u FROM CandidateUser u WHERE :mail = u.primaryEmail")
```

Comment:
We have decided to have the primaryEmail included in the mailList so you can set this one back to MEMBER OF